tools/c7n_gcp/c7n_gcp/resources/sql.py

```diff
@@ -159,11 +154,28 @@ class resource_type(ChildTypeInfo):
         }
 
         @staticmethod
-        def get(client, resource_info):
-            parameters = SqlInstanceChildWithSelfLink._get_base_query_parameters(resource_info)
-            parameters['id'] = resource_info['backup_run_id']
+        def get(client, event):
+            project = jmespath.search('protoPayload.response.targetProject', event)
+            instance = jmespath.search('protoPayload.response.targetId', event)
+            insert_time = jmespath.search('protoPayload.response.insertTime', event)
+            parameters = {'project': project,
+                          'instance': instance,
+                          'id': SqlBackupRun.resource_type.from_insert_time_to_id(insert_time)}
             return client.execute_command('get', parameters)
 
+        @staticmethod
+        def from_insert_time_to_id(insert_time):
+            """
+            Backup Run id is not available in a log record directly.
+            Fortunately, as it is an integer timestamp representation,
+            it can be retrieved by converting raw insert_time value.
+
+            :param insert_time: a raw timestamp to convert to id
+            :return: an integer representing the raw timestamp passed
+            """
+            delta = parse(insert_time).replace(tzinfo=None) - datetime.utcfromtimestamp(0)
```

Comment:
please identify the types in the doc string or inline comments, ie. insert_time is a utc isoformatted date time string, return value is integer microseconds since unix epoch.

Reply:
Thank you for the suggestion, improved the docs.